app/modules/collaborations/models.py

```diff
@@ -221,11 +232,47 @@ def notify_pending_users(self):
                     )
 
     def _notify_user(self, sending_user_assoc, receiving_user_assoc, notification_type):
-        from app.modules.notifications.models import Notification, NotificationBuilder
+        from app.modules.notifications.models import (
+            Notification,
+            NotificationBuilder,
+            NotificationType,
+        )
 
         builder = NotificationBuilder(sending_user_assoc.user)
         builder.set_collaboration(self)
-        Notification.create(notification_type, receiving_user_assoc.user, builder)
+        notif = Notification.create(notification_type, receiving_user_assoc.user, builder)
+        # I expect this to be set by the builder but it sometimes does not, unsure why.
```

Comment:
It really should have been. Curious about when you were seeing it unset, was it when running uit tests?

Reply:
Glad you double checked, I thought one of the tests broke without that line but was not able to replicate just now. Must have been one of those debugging red herrings, so I'm removing this line.